Packs/XMCyber/ReleaseNotes/1_0_6.md

```diff
@@ -0,0 +1,8 @@
+
+#### Incident Types
+- **XM Cyber IP Incident**
+An incident type for incoming incidents that only have an IP address
+
+#### Playbooks
+- **IP Enrichment - XM Cyber**
+Bug fix
```

Comment:
- Fixed an issue where the playbook failed when the entity wasn't found in XM Cyber.